src/main/java/com/spring/studentmanagement/controllers/ErrorController.java

```diff
@@ -29,4 +29,15 @@ public String getUnauthorizedView(Model model) {
         model.addAttribute("errorTitle", "Unauthorized");
         return "errors/error-401";
     }
+
+    @GetMapping(
+            path = "/error-403",
+            produces = TEXT_HTML_VALUE,
+            headers = {ACCEPT, ACCEPT_ENCODING, ACCEPT_LANGUAGE, CACHE_CONTROL, CONNECTION, HOST, REFERER, USER_AGENT}
+    )
+    @ResponseStatus(value = HttpStatus.UNAUTHORIZED)
```

Comment:
**HttpStatus.UNAUTHORIZED?** 
Please adapt the code according business requirements and not just copy paste.
HttpStatus unauthorized code is 401 and has nothing to do with status code 403 which is FORBIDDEN